### ISSUE TITLE
Frequent crash when logging onto RandomBot Character

### ISSUE THREAD

**Issue by Dreathean** [liyunfan1223/mod-playerbots#16]
Haven't been able to determine the cause, but shortly after logging onto any randombot character, the server crashes in <1 minute. Tested on multiple accounts/characters

**Comment by garfieldz90** [comment]
problem solved i think, not frequent crash anymore.

**Comment by Dreathean** [comment]
Tested and this is not solved. To be clear, I'm not talking about a crash occurring when on your character. I'm talking about the server crashing after logging into one of the Randombot generated accounts/characters (rndbot1, etc). Logging onto one of those characters always crashes the server <30 seconds

**Comment by Dreathean** [comment]
Fair enough, I'll re-open it! Out of curiosity what use cases are you needing to login to a live bot? I assume you're referring to player-created bots and not the randomly generated ones right? Otherwise why not remove the bot off that character temporarily when you need to login to it?
